<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSettingsTemplates</class>
 <widget class="QWidget" name="WdgSettingsTemplates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Template:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboTemplate"/>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="cmdEdit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="text">
            <string>Edit</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="arrowType">
            <enum>Qt::NoArrow</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="cmdAdd">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="cmdDelete">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
           <property name="icon">
            <iconset resource="../res.qrc">
             <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabHeader">
      <attribute name="title">
       <string>Header</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTextEdit" name="txtHeader"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabEnum">
      <attribute name="title">
       <string>Enum</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTextEdit" name="txtEnum"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStruct">
      <attribute name="title">
       <string>Struct</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTextEdit" name="txtStruct"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabClass">
      <attribute name="title">
       <string>Class</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTextEdit" name="txtClass"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMethod">
      <attribute name="title">
       <string>Method</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QTextEdit" name="txtMethod"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cmdSave">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdAdd</sender>
   <signal>clicked()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>addTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>674</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdEdit</sender>
   <signal>clicked()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>editTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdDelete</sender>
   <signal>clicked()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>removeTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboTemplate</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>templateIdxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdSave</sender>
   <signal>clicked()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>saveTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtHeader</sender>
   <signal>textChanged()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>contentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtClass</sender>
   <signal>textChanged()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>contentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtEnum</sender>
   <signal>textChanged()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>contentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtMethod</sender>
   <signal>textChanged()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>contentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtStruct</sender>
   <signal>textChanged()</signal>
   <receiver>WdgSettingsTemplates</receiver>
   <slot>contentChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addTemplate()</slot>
  <slot>removeTemplate()</slot>
  <slot>saveTemplate()</slot>
  <slot>editTemplate()</slot>
  <slot>templateIdxChanged(int)</slot>
  <slot>contentChanged()</slot>
  <slot>templatesModified()</slot>
 </slots>
</ui>
